about.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;500;600;700;900&display=swap" rel="stylesheet">
+  
+
+  <link rel="stylesheet" href="styles-conference/stylesheets/styles.css">
+  <title>Swing Dance Styles and Steps Library</title>
+</head>
+<body>
+  <header>
+    <div class="popup-menu invisible">
+      <img class="close-button" src="styles-conference/assets/Icon-Cancel.svg" alt="">
+      <ul class="popup-menu-list">
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="./index.html#program">Program</a></li>
+        <li><a href="#AJ">Join</a></li>
+        <li><a href="./index.html#sponsors">Sponsor</a></li>
+        <li><a href="#news">News</a></li>
+        <li><button class="sdssl-button">SDSSL Campaign</button></li>
+      </ul>
+    </div>
+    <nav>
+      <div class="dark-header">
+        <div class="media">
+          <a href="https://www.facebook.com/martinezequiel.gonzalez.5"><img src="styles-conference/assets/fb_icon.png" alt="facebook_icon"></a>
+          <a href="https://twitter.com/MarezegonZ"><img src="styles-conference/assets/tw_icon.png" alt="twitter_icon"></a>
+        </div>
+        <div class="dark-menu-list">
+          <a href="#">Logout</a>
+          <a href="#">Mypage</a>
+          <a href="#">Spanish</a>
+        </div>       
+      </div>
+      <div class="navbar toggle-selection">
+        <a class="ham-link"href="#">
+          <img class= "ham-img" src="styles-conference/assets/ham.png" alt="">
+        </a>
+        <a href="./index.html"><img class="logo-navbar" src="styles-conference/assets/sd_logo.png" alt="logo_sd"></a>
+        <ul class="menu-list-desktop">
+          <li><a class="home-about" href="./index.html">Home</a></li>
+          <li><a class="menu-item-desktop" href="./index.html#program">Program</a></li>
+          <li><a class="menu-item-desktop" href="./index.html#AJ">Join</a></li>
+          <li><a class="menu-item-desktop" href="#sponsors">Sponsor</a></li>
+          <li><a class="menu-item-desktop" href="#news">News</a></li>
+          <li><button class="sdssl-button">SDSSL Campaign</button></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <section id="about" class="about toggle-blur">
+    <section class="about-page about1">
+      <div class="headings">
+        <h2 class="about-hello">"Hello! Swing Dancers"</h2>
+        <br>
+        <h1 class="about-title">SWING DANCE STYLES AND STEPS LIBRARY 2023</h1>

```

Comment:
- Good job adding at least one h1 to your page, However while doing it you have put it below an h2 tag which is not recommended according to the semantic HTML rules :recycle: